shopfloor_reception_mobile/static/src/scenario/reception_states.js

```diff
@@ -240,6 +240,7 @@ export const reception_states = function () {
                         picking_id: this.state.data.picking.id,
                         selected_line_id: this.line_being_handled.id,
                         location_name: location.text,
+                        // FIXME if it is always set to true, it is not really used ?
```

Comment:
don't we fix it here?

Reply:
Actually not. This is not the same endpoint that is called. So this one has not been changed.
Because the frontend always return true, I did not see the point of updating it, maybe the question is should it be removed ?